react/AvailabilitySubscriber.tsx

```diff
@@ -160,9 +163,13 @@ function AvailabilitySubscriber(props: Props) {
             <Input
               name="name"
               type="text"
-              placeholder={intl.formatMessage({
-                id: 'store/availability-subscriber.name-placeholder',
-              })}
+              placeholder={
+                props.namePlaceholder
+                  ? props.namePlaceholder
+                  : intl.formatMessage({
+                      id: 'store/availability-subscriber.name-placeholder',
+                    })
```

Comment:
Move this to a variable outside the markup. And this conditional can also be simplified to something like:

```js
props.namePlaceholder || intl.formatMessage({...})
```